MdePkg/Include/IndustryStandard/PciExpress21.h

```diff
@@ -386,7 +416,7 @@ typedef struct {
   PCI_REG_PCIE_LINK_CAPABILITY2      LinkCapability2;
   PCI_REG_PCIE_LINK_CONTROL2         LinkControl2;
   PCI_REG_PCIE_LINK_STATUS2          LinkStatus2;
-  UINT32                             SlotCapability2;
+  PCI_REG_PCIE_SLOT_CAPABILITY2      SlotCapability2;

```

Comment:
This change will impact the source code. Have you meet with the real case with this field?

Reply:
Yes, this change is essential to support PciE Generation 6 specification. And it is inevitable.

Reply:
The original field was intended as a placeholder till the capability was defined in the specification.